c2h/generators.cu

```diff
@@ -620,19 +650,16 @@ struct vec_gen_t
   }
 };
 
-#define VEC_GEN_MOD_SPECIALIZATION(VEC_TYPE, SCALAR_TYPE)                                                        \
-  template <>                                                                                                    \
-  void gen<VEC_TYPE>(modulo_t mod, c2h::device_vector<VEC_TYPE> & data)                                          \
-  {                                                                                                              \
-    thrust::tabulate(c2h::device_policy, data.begin(), data.end(), vec_gen_t<VEC_TYPE, SCALAR_TYPE>{mod.get()}); \
-  }
+#  define VEC_GEN_MOD_SPECIALIZATION(VEC_TYPE, SCALAR_TYPE)                                                        \
+    template <>                                                                                                    \
+    void gen<VEC_TYPE>(modulo_t mod, c2h::device_vector<VEC_TYPE> & data)                                          \
+    {                                                                                                              \
+      thrust::tabulate(c2h::device_policy, data.begin(), data.end(), vec_gen_t<VEC_TYPE, SCALAR_TYPE>{mod.get()}); \
+    }
 
 VEC_GEN_MOD_SPECIALIZATION(short2, short);
-
 VEC_GEN_MOD_SPECIALIZATION(uchar3, unsigned char);
-
 VEC_GEN_MOD_SPECIALIZATION(ulonglong4, unsigned long long);
-
 VEC_GEN_MOD_SPECIALIZATION(ushort4, unsigned short);
-
+#endif
```

Comment:
Missing comment

Reply:
I probably have a lot more of those! It's still a draft.